superflore/generators/bitbake/run.py

```diff
@@ -58,6 +58,12 @@ def main():
     if not args.dry_run:
         if 'SUPERFLORE_GITHUB_TOKEN' not in os.environ:
             raise NoGitHubAuthToken()
+        err('Generating bitbake recipes without specifying --dry-run is not '
```

Comment:
Move to be before the test for `SUPERFLORE_GITHUB_TOKEN`.

Actually, since we now require `--dry-run`, this should be change to an "assert" of faulty program logic.

Reply:
We can delete all this, because now people won't see this longer explanation about --dry-run, only:

Deploy ROS packages into OpenEmbedded Linux: error: the following arguments are required: --dry-run